source/main/gameplay/OutProtocol.cpp

```diff
@@ -169,9 +169,9 @@ bool OutProtocol::Update(float dt, Actor* truck)
         gd.DashLights |= DL_SIGNAL_L;
         gd.DashLights |= DL_SIGNAL_R;
         gd.DashLights |= DL_SIGNAL_ANY;
-        if (truck->tc_present)
+        if (!truck->tc_nodashboard)

```

Comment:
isn't this `tc_nodash` ? 😮

Reply:
AppVeyor replies: `cl : Command line warning D9025 : overriding '/W3' with '/w'
source\main\gameplay\OutProtocol.cpp(172): error C2039: 'tc_nodashboard': is not a membe
r of 'Actor'`